pkg/k8s/node.go

```diff
@@ -138,6 +111,40 @@ func ParseNode(k8sNode *slim_corev1.Node, source source.Source) *nodeTypes.Node
 			}
 		}
 	}
+
+	newNode.Labels = k8sNode.GetLabels()
+
+	if !option.Config.AnnotateK8sNode {
+		return newNode
+	}
+
+	// Any code bellow this line will depend on k8s node annotations. If we are
+	// not annotating the node then we should use any annotation.
```

Comment:
```suggestion
	// not annotating the node then we should not use any annotation.
```

I think that's what you mean?